kickstarter/android-oss
# NTV-208: Scrolling transition of the video player

# 📲 What

Create scrolling animation for media header.

# 🤔 Why

Project page revamp.

# 🛠 How

Created a motion layout and motion scene that listens to tab clicks, and starts/stops the animation upon accordingly. Also created a collapsing toolbar for the scroll behavior on the view pager. 

# 👀 See

https://user-images.githubusercontent.com/19390326/142504477-112dc84b-54f9-486d-ad02-e5ac9d0aee4e.mp4

# 📋 QA

- Open a project
- Tap on a tab, media header should scroll out of view
- Return to the overview tab. Media header should remain out of view
- Scroll down on overview tab, media header should animate into view
- In the overview tab, swipe up to scroll. Video should scroll inline, tool bar should be pinned to the top.

# Story 📖

[NTV-208: Scrolling transition of the video player](https://kickstarter.atlassian.net/browse/NTV-208)
